hdfs_datanode/datadog_checks/hdfs_datanode/data/conf.yaml.example

```diff
@@ -69,103 +98,163 @@ instances:
+    ##
+    ##   - basic
+    ##     |__ username
+    ##     |__ password
+    ##   - digest
+    ##     |__ username
+    ##     |__ password
+    ##   - ntlm
+    ##     |__ ntlm_domain
+    ##     |__ password
+    ##   - kerberos
+    ##     |__ kerberos_auth
+    ##     |__ kerberos_cache
+    ##     |__ kerberos_delegate
+    ##     |__ kerberos_force_initiate
+    ##     |__ kerberos_hostname
+    ##     |__ kerberos_keytab
+    ##     |__ kerberos_principal
+    ##   - aws
+    ##     |__ aws_region
+    ##     |__ aws_host
+    ##     |__ aws_service
+    ##
+    ## The `aws` auth type relies on boto3 to automatically gather AWS credentials (e.g. from `.aws/credentials`).
+    ## Details: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials
     #
-    # auth_type: <AUTH_TYPE>
+    # auth_type: basic
 
     ## @param username - string - optional
-    ## The username to use if services are behind basic auth.
+    ## The username to use if services are behind basic or digest auth.
     #
     # username: <USERNAME>
 
-    ## @param ntlm_domain - string - optional
-    ## If your services uses NTLM authentication, you can optionally
-    ## specify a domain that will be used in the check. For NTLM Auth,
-    ## append the username to domain, not as the `username` parameter.
-    ## Example: example_ntlm_domain\example_username
-    #
-    # ntlm_domain: <DOMAIN>
-
     ## @param password - string - optional
     ## The password to use if services are behind basic or NTLM auth.
     #
     # password: <PASSWORD>
 
+    ## @param ntlm_domain - string - optional
+    ## If your services use NTLM authentication, you can specify
+    ## a domain that is used in the check. For NTLM Auth, append
+    ## the username to domain, not as the `username` parameter.
+    #
+    # ntlm_domain: <NTLM_DOMAIN>/<USERNAME>
+
     ## @param kerberos_auth - string - optional - default: disabled
-    ## If your service uses Kerberos authentication, you can specify the Kerberos
+    ## If your services use Kerberos authentication, you can specify the Kerberos
     ## strategy to use between:
-    ##  * required
-    ##  * optional
-    ##  * disabled
+    ##
+    ##   - required
+    ##   - optional
+    ##   - disabled
     ##
     ## See https://github.com/requests/requests-kerberos#mutual-authentication
     #
     # kerberos_auth: disabled
 
+    ## @param kerberos_cache - string - optional
+    ## Sets the KRB5CCNAME environment variable.
+    ## It should point to a credential cache with a valid TGT.
+    #
+    # kerberos_cache: <KERBEROS_CACHE>
+
     ## @param kerberos_delegate - boolean - optional - default: false
-    ## Set to true to enable kerberos delegation of credentials to a server that requests delegation.
+    ## Set to `true` to enable kerberos delegation of credentials to a server that requests delegation.
```

Comment:
```suggestion
    ## Set to `true` to enable Kerberos delegation of credentials to a server that requests delegation.
```

Reply:
This line comes from the template `instances/http` https://github.com/DataDog/integrations-core/pull/6900/files#diff-1a1e97c195f87173eaceb80d31c30af3R24

https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L116

It is used by multiple integrations, so we should do it in a separate PR to sync all the integrations